python/src/pyx/logs_db/db_mysql.py

```
@@ -1,134 +1,177 @@
-        conn = get_connection()
-        cursor = conn.cursor()
+        with get_connection() as (_, cursor):
+            if many:
+                cursor.executemany(query, params)
+            else:
+                cursor.execute(query, params)
+
+            cursor.connection.commit()
+            return True
+    except MySQLError as e:
+        logger.error(f"Database query error: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)